### ISSUE TITLE
Reconsider the move to Java 11 in favour of Java 8

### ISSUE THREAD

**Issue by caprica** [caprica/vlcj#949]
Dropping 1.6 is fair enough.

Moving to Java 11 brought with it the module system which, frankly, causes many pain points.

Consider going back to Java 8 as the project baseline, the module-info artefacts can remain, but the project should build against Java 8.

Related:
#862 
#928 
#930
#944

Also would remove the burden of maintaining the 4.4.x series along with a mirrored 4.5.x series - i.e. the 4.5.x series would no longer be required and could be written off as a folly.

To summarise currently:

4.4.x branch is 1.6 (and still current)
4.x branch branch is 11 and modules (and current of course, 4.5.x is the latest)
5.x master is in development and is currently 11 and modules

The consideration therefore is for the 5.x master to go back to Java 8 and not care about modules (although it should be harmless to retain the module system artefacts? I am not sure about that).

**Comment by caprica** [comment]
Maybe follow this:
https://www.oxymorus.com/maven-package-to-multiple-java-versions/

**Comment by caprica** [comment]
Will need some solution for handling module-info.java since it can not be included in a 1.8 build.

**Comment by caprica** [comment]
May be interesting:
https://blog.codefx.org/tools/multi-release-jars-multiple-java-versions/

**Comment by caprica** [comment]
Possibly helpful:
https://github.com/hboutemy/maven-jep238

**Comment by caprica** [comment]
Reverting all of this. It's too much burden for too little benefit.

 - Baseline is now Java 1.8.
 - No modules system.
 - The 4.4.x series is Java 1.6, 4.4.2 will be the last version pending any critical bugs (very unlikely)
 - The 4.5.x series (Java 11 and modules) is abandoned
 - Ongoing 5.x development will move to 1.8 (was Java 11 and modules)

Life is now much easier.

**Comment by caprica** [comment]
This has all now been cleaned up and modules ditched.

I kept automatic module name for the jar manifest.
